src/client/public/css/pages/home.css

```diff
@@ -1,3 +1,17 @@
+.heading {
+	font-size: 250%;
+}
```

Comment:
Font size should not be in percents.
Use px variable in base css file for defining font size.
Make sure that the font size corresponds to that in the design